app/controllers/concerns/api/users_controller.rb

```diff
@@ -21,5 +21,11 @@ def show
     def me
       render :show
     end
+
+    def unpublish
+      authorize(@user, :unpublish_all_articles?)
```

Comment:
Reusing `:unpublish_all_articles?` that already existed from the Admin dashboard implementation